javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,37 @@
+console.log('Script loaded');
+let products = [];
+
+const testProductNames = ['Flat screen', 'Mobile phone', 'Wallet'];
+
+function renderProducts() {
+    const productsAvailable = getAvailableProducts();
+    const ulTag = document.querySelector("section > ul");
+
+    for (let i = 0; i < productsAvailable.length; i++) {
+        const liTag = document.createElement('li');
+        const childUlTag = document.createElement("ul");
+        for (let j = 1; j <= 4; j++) {
```

Comment:
This is good, and works well. But it's hard to understand what is happening without knowing what the data looks like, it's also referencing the object values by index, is a bad way to access object values.  The best way to access object values is by name for example `productsAvailable['price']`

When dealing with objects like this you should use object key iteration like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in it will make the code clearer and better